README.md

```diff
@@ -1,2 +1,29 @@
-# .github
-default-community-health
+# 18F Project Leadership Collective
+
+The purpose of this repository is to provide ability to submit ideas for talks or resources to the 18F Project Leadership Collective.
+
+* **[Submit a talk]()**
+* **[Request a resource]()**
+
+## About the Collective
+
+The Project Leadership Collective (PLC) provides 18F staff an opportunity to share lessons learned, successes, new methods, and know how related to leading 18F project teams.
+
+Project Leadership is the practice of coordinating and representing a project, both to the partner and within 18F. This collective is to help us learn from one each other about the practices and methods that help ensure project success.
+
+## Goals
+
+* Share lessons learned between 18F staff on project leadership
+* Identify and share best practices and new methods
+* Learn about organizational initiatives and information that a lead is responsible for: Things like: Project Impact, Project Health, and other forthcoming initiatives
+* Capture feedback on organizational initiatives, ensuring they improve over time and have the intended impact for 18F and our partners
+* Improve your practice as a lead
+* Identify and ask about concerns, questions and ideas in a safe space outside of your project team
+* Surface patterns across project teams and share bottom-up knowledge necessary during handoffs between teams, term-outs, and other changes
+* Support the continuity of institutional knowledge
+
+## Practice
+
+* We conduct monthly meetings open to the 18F community 
+* We maintain the Project Leadership Toolkit
+* We build and maintain resources related to project leadership
```

Comment:
What do you think of adding the link here?

* We build and maintain [resources related](https://handbook.tts.gsa.gov/18f/projects-partners/leading-projects/) to project leadership

Reply:
done!